nncf/experimental/torch/fx/transformations.py

```diff
@@ -952,6 +952,28 @@ def merge_linear_and_bias(model: torch.fx.GraphModule):
     _merge_node_and_bias(model, _is_linear)
 
 
+def _get_connected_nodes(graph: torch.fx.Graph) -> List[torch.fx.Node]:
+    """
+    Returns the List of nodes which are directly or indirectly connected
```

Comment:
What do indirectly connected nodes mean?

Reply:
For example:
```
node1   node2
    |             |
       node3
           |
        output
```
In the case above, node1 and node2, are considered to be indirectly connected to the output node.